[2025-03-02T07:50:52.943+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-02T07:50:52.948+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_price_daily_load.fetch_and_load_data manual__2025-03-02T07:50:51.344787+00:00 [queued]>
[2025-03-02T07:50:52.951+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_price_daily_load.fetch_and_load_data manual__2025-03-02T07:50:51.344787+00:00 [queued]>
[2025-03-02T07:50:52.951+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-03-02T07:50:52.955+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): fetch_and_load_data> on 2025-03-02 07:50:51.344787+00:00
[2025-03-02T07:50:52.959+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'stock_price_daily_load', 'fetch_and_load_data', 'manual__2025-03-02T07:50:51.344787+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/daily_stock_price.py', '--cfg-path', '/tmp/tmpzvghjhbw']
[2025-03-02T07:50:52.960+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=259) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-02T07:50:52.961+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask fetch_and_load_data
[2025-03-02T07:50:52.961+0000] {standard_task_runner.py:72} INFO - Started process 262 to run task
[2025-03-02T07:50:52.977+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_price_daily_load.fetch_and_load_data manual__2025-03-02T07:50:51.344787+00:00 [running]> on host 7f84c8d31b44
[2025-03-02T07:50:53.005+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stock_price_daily_load' AIRFLOW_CTX_TASK_ID='fetch_and_load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-02T07:50:51.344787+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-02T07:50:51.344787+00:00'
[2025-03-02T07:50:53.005+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-02T07:50:53.018+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-03-02T07:50:53.019+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-02T07:50:53.019+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-02T07:50:53.780+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T07:50:53.782+0000] {logging_mixin.py:190} INFO - YF.download() has changed argument auto_adjust default to True
[2025-03-02T07:50:54.351+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T07:50:55.472+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T07:50:55.473+0000] {logging_mixin.py:190} INFO - ✅ Inserted 123 new rows for TSLA.
[2025-03-02T07:50:55.782+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T07:50:56.172+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T07:50:57.150+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T07:50:57.151+0000] {logging_mixin.py:190} INFO - ✅ Inserted 123 new rows for SPY.
[2025-03-02T07:50:57.152+0000] {connection.py:788} INFO - closed
[2025-03-02T07:50:57.199+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2025-03-02T07:50:57.256+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-02T07:50:57.272+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-02T07:50:57.273+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_price_daily_load, task_id=fetch_and_load_data, run_id=manual__2025-03-02T07:50:51.344787+00:00, execution_date=20250302T075051, start_date=20250302T075052, end_date=20250302T075057
[2025-03-02T07:50:57.305+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-02T07:50:57.317+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-02T07:50:57.318+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
